From 01878e3a5bca6c0d1e57b97c2588ef9efeb128c6 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Thu, 27 Dec 2018 18:24:31 +0200
Subject: [PATCH 0835/1051] dts: a3700: enable dma coherence for PCIE interface

Change-Id: Ifbf219f41612940f22df23c88794542949005195
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/62150
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Liron Himi <lironh@marvell.com>
(cherry picked from commit 5b4a318b64a9b710acc1bb4b569f1392185bfcc0)
Reviewed-on: http://vgitil04.il.marvell.com:8080/62388
Reviewed-on: https://sj1git1.cavium.com/1565
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index e1b4d50d95b6..a7279b78904d 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -434,6 +434,7 @@
 			#interrupt-cells = <1>;
 			msi-parent = <&pcie0>;
 			msi-controller;
+			dma-coherent;
 			ranges = <0x82000000 0 0xe8000000   0 0xe8000000 0 0x1000000 /* Port 0 MEM */
 				  0x81000000 0 0xe9000000   0 0xe9000000 0 0x10000>; /* Port 0 IO*/
 			interrupt-map-mask = <0 0 0 7>;
-- 
2.17.1

